keras_hub/src/models/retinanet/retinanet_label_encoder.py

```diff
@@ -193,11 +178,12 @@ def _encode_sample(self, gt_boxes, gt_classes, anchor_boxes, image_shape):
             matched_gt_boxes, (-1, ops.shape(matched_gt_boxes)[1], 4)
         )
 
-        box_target = _encode_box_to_deltas(
+        box_target = encode_box_to_deltas(
             anchors=anchor_boxes,
             boxes=matched_gt_boxes,
             anchor_format=self.bounding_box_format,
             box_format=self.bounding_box_format,
+            encoding_format=self.encoding_format,
             variance=self.box_variance,
             image_shape=image_shape,
         )
```

Comment:
for the return statement of this method. assign the value to `box_targets` and `class_targets` and then return them for clarity